src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -102,18 +108,38 @@ export default class EthereumBlockchainService implements BlockchainService {
 
         logger.imp("Transaction data:" + JSON.stringify(txData));
 
+        logEvent.ethereum({
+          type: 'txRequest',
+          ...txData
+        });
         const txResponse: providers.TransactionResponse = await wallet.sendTransaction(txData);
-
+        logEvent.ethereum({
+          type: 'txResponse',
+          ...txResponse
+        });
         const caip2ChainId = "eip155:" + txResponse.chainId;
         if (caip2ChainId != this.chainId) {
           // TODO: This should be process-fatal
           throw new Error("Chain ID of connected blockchain changed from " + this.chainId + " to " + caip2ChainId)
         }
 
         const txReceipt: providers.TransactionReceipt = await this.provider.waitForTransaction(txResponse.hash);
+        logEvent.ethereum({
+          type: 'txReceipt',
+          ...txReceipt
+        });
         const block: providers.Block = await this.provider.getBlock(txReceipt.blockHash);
 
-        logger.imp(`Transaction successfully written to Ethereum ${network} network. Transaction hash ${txReceipt.transactionHash}`);
+        const status = txReceipt.byzantium ? txReceipt.status : -1;
+        let statusMessage = (status == TX_SUCCESS) ? 'success' : 'failure';
+        if (!txReceipt.byzantium) {
+          statusMessage = 'unknown';
+        }
+        logger.imp(`Transaction completed on Ethereum ${network} network. Transaction hash: ${txReceipt.transactionHash}. Status: ${statusMessage}.`);
```

Comment:
where do these diagnostic logs against the `logger` go if we're in log-to-files mode?  I didn't see a log path specified for them above, but maybe it's handled somewhere else I missed?

Reply:
No didn't include that. I'll add it though